packages/fxa-auth-server/bin/key_server.js

```diff
@@ -112,13 +112,11 @@ async function run(config) {
 
   try {
     await server.start();
-    log.info({
-      op: 'server.start.1',
+    log.info('server.start.1', {
```

Comment:
Should other log calls be updated in this file?

Looks like most others in this file follow the old pattern (e.g [like here during shutdown](https://github.com/mozilla/fxa/blob/9589fc7906810c085ce755852f42804476dca8e5/packages/fxa-auth-server/bin/key_server.js#L140)) and that doesn't seem to match [the mozlog module docs](https://www.npmjs.com/package/mozlog#usage).

Reply:
Good catch.  I thought these two calls were from recent changes, but they were probably just formatting changes?  I'll update the other log calls.